heroku/cli
# refactor(pg-v5): Move command pg:maintenance:run to oclif

### Description
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001bCpspYAC/view

This converts `pg:maintenance:run` to oclif and updates its associated tests.

### Testing

1) Pull down this branch
2) Run `./bin/run pg:maintenance -a APP` with a maintenance t orun
3) Verify the command succeeds, running the maintenance
4) Compare with prod
